raiden/api/v1/resources.py

```diff
@@ -121,3 +122,15 @@ def get(self, **kwargs):
             kwargs['from_block'],
             kwargs['to_block']
         )
+
+
+class TokenSwapsResource(BaseResource):
+
+    put_schema = TokenSwapsSchema()
+
+    def __init__(self, **kwargs):
+        super(TokenSwapsResource, self).__init__(**kwargs)
+
+    @use_kwargs(put_schema)
+    def put(self, **kwargs):
```

Comment:
Hmmm why? I was using explicit names before. But now since they match the function inside the corresponding `rest.py` I thought it's nicer to have `**kwargs`.

You don't like the fact that you can't see the exact arguments when seeing this file? And that if at any point anything new is added to the schema we will get an exception due to the args mismatch?

Hmm now that I write it down myself .... yeah not a good idea to use `**kwargs`